src/main/scala/cromwell/engine/ExecutionTable.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.engine
+
+import cromwell.binding.{Call, WdlBinding}
+
+/**
+ * Corresponds to the "execution table" of our discussions.
+ */
```

Comment:
I've thought about it orders of magnitude less long than you all so take it w/ a grain of salt but for the symbol stuff my first thought was just Symbols (which contains a colleciton of Symbol).

I have fewer amazing suggestions for execution, and by fewer I mean 0. I like ExecutionState